src/raster/r.sun.daily/r.sun.daily.py

```diff
@@ -431,13 +441,27 @@ def check_daily_map_names(basename, mapset, start_day, end_day, day_step):
             )
 
 
-def sum_maps(sum_, basename, suffixes):
+def maps_sum(out_, basename, suffixes):
     """
     Sum up multiple raster maps
     """
     maps = "+".join([basename + suf for suf in suffixes])
     grass.mapcalc(
-        "{sum_} = {new}".format(sum_=sum_, new=maps), overwrite=True, quiet=True
+        "{out_} = {new}".format(out_=out_, new=maps), overwrite=True, quiet=True
+    )
+
+
+def maps_avg(out_, basename, suffixes):
+    """
+    Get average value from multiple raster maps.
+    """
+    maps = "+".join([basename + suf for suf in suffixes])
+    grass.mapcalc(
+        "{out_} = ({new}) / {maps_count}".format(
```

Comment:
consider f-string here

Reply:
I can't blame you, but you are introducing new code here and there is no need to be consistent with old practices.

Reply:
I believe that it is clearer to keep the full-module practice and then change the practice in the whole module in another commit, but whatever. It seems like a minor nitpick aspect to me, so I don't feel any need to fight for that tooth and nail. Changed in 961aa5b62993f8ab4959c87d4a7b25c78faa941e.